newIDE/app/src/UI/Theme/index.js

```diff
@@ -90,7 +90,8 @@ const rtlOverrides = {
   },
 };
 
-export const defaultTheme: ActualTheme = {
-  gdevelopTheme: DefaultTheme.gdevelopTheme,
-  muiTheme: createLtrTheme(DefaultTheme.muiThemeOptions),
-};
+export const defaultTheme: ActualTheme = (() => {
+  const theme = DefaultTheme;
+  theme.muiThemeOptions = createLtrTheme(DefaultTheme.muiThemeOptions);
+  return theme;
```

Comment:
Are you sure this is necessary? This will mutate `DefaultTheme`, which sounds dangerous to me.

Reply:
Yikes, good catch. Yeah, let's not do that.  
Updated to make a copy with `muiThemeOptions` overridden.